scss/civicrm/common/_bootstrap-calendar.scss

```diff
@@ -1,124 +1,122 @@
-.page-civicrm {
```

Comment:
why was `page-civicrm` required at all?

Reply:
The only reason the `page-civicrm` class is required is to scope the datepicker styles to civicrm pages only otherwise we never need it.